crates/cargo-test-support/src/compare.rs

```diff
@@ -96,6 +97,121 @@ pub fn assert_ui() -> snapbox::Assert {
         .redact_with(subs)
 }
 
+/// Assertion policy for functional end-to-end tests
+///
+/// This emphasizes showing as much content as possible at the cost of more brittleness
+///
+/// # Snapshots
+///
+/// Updating of snapshots is controlled with the `SNAPSHOTS` environment variable:
+///
+/// - `skip`: do not run the tests
+/// - `ignore`: run the tests but ignore their failure
+/// - `verify`: run the tests
+/// - `overwrite`: update the snapshots based on the output of the tests
+///
+/// # Patterns
+///
+/// - `[..]` is a character wildcard, stopping at line breaks
```

Comment:
Guess we might want to add `[CWD]` in the future. Nevertheless it is not a blocker.